tests/platforms/linux/appimage/test_mixin.py

```diff
@@ -102,13 +102,13 @@ def test_verify_non_linux_no_docker(tmp_path):
         base_path=tmp_path / "base_path",
         data_path=tmp_path / "briefcase",
     )
-    command.tools.host_os = "WeirdOS"
+    command.tools.host_os = "Darwin"
     command.use_docker = False
 
     # Verify the Docker tool
     with pytest.raises(
         BriefcaseCommandError,
-        match="Linux AppImages can only be generated on Linux without Docker",
```

Comment:
It's not being picked up by coverage, but both Darwin and WeirdOS should be tested here.

Reply:
Discussed in [comment](https://github.com/beeware/briefcase/pull/1066#issuecomment-1411196292).